README.md

```diff
@@ -187,35 +147,37 @@ highlight link multiple_cursors_visual Visual
 ## FAQ
 
 #### **Q** <kbd>CTRL</kbd>+<kbd>n</kbd> doesn't seem to work in gVIM?
-
 **A** Try setting `set selection=inclusive` in your `~/.gvimrc`
 
-#### **Q** How can I select `n` keywords with several keystrokes? I have tried `200<C-n>` which does not work.
+#### **Q** is it also working on Mac?
+**A** On Mac OS, [MacVim](https://code.google.com/p/macvim/) is known to work.
 
+#### **Q** How can I select `n` keywords with several keystrokes? `200<C-n>` which does not work...
```

Comment:
Maybe it's me, but now when I read this Q&A it feels like the A hasn't really answered the Q. Maybe here should be two different Q&As, e.g.:
Q: How do I do `200<C-n>`?
A: Do it like this... `<demo>`

Q: How do I convert my search results into cursors?
A: Use `MultipleCursorsFind` and a mapping like this: ...

Reply:
This is a workaround to the fact that we cannot do ```200<C-n>```, so I think this is appropriate.
If you wanted to split it in 2 questions like you did, the ansers would be:
```
Q: How do I do ```200<C-n>```?
A: You cannot.

Q: How do I convert my search results into cursors?
A: see gif example in About section.
```
which is not super useful. What do you think ?
For info I just copy/pasted previous Q&A, but I kind of like the answer like it is today.

Reply:
I think it's not a bad idea to state explicitly that `200<C-n>` is not supported right now, but suggest the user another way that he/she can use to achieve similar result.

> A: see gif example in About section.

The gif example demonstrates how to use regexps in command line, and example here demonstrates a mapping. Giving such detailed examples may be too obvious for some experienced vim users, but on the other hand I think there are novice users who could benefit from it. Thinking about it more, I'd add the example from the gif to the docs as well.

Sure, the remark is not about your edits, it's just only now that I thought about this particular Q&A and it sounded a bit "dodgy" to me. If you think it's OK, it's fine with me!